samples/batch/transcription-enabled-storage/CreateTranscription/CreateTranscription.cs

```diff
@@ -0,0 +1,295 @@
+                        log.LogError($"Failed with FormatException. Write message for {jobName} to report file.");
+                        var errorTxtName = jobName + "_error.txt";
+                        var responseMessage = $"Transcription failed with FormatException:\n{e.Message}";
+
+                        var errorOutputContainer = CreateTranscriptionEnvironmentVariables.ErrorReportOutputContainer;
+                        await StorageUtilities.WriteTextFileToBlobAsync(CreateTranscriptionEnvironmentVariables.AzureWebJobsStorage, responseMessage, errorOutputContainer, errorTxtName, log).ConfigureAwait(false);
+
+                        return string.Empty;
+                    }
+                    catch (TimeoutException e)
+                    {
+                        log.LogError($"Timeout while creating post, re-enqueueing transcription create. Message: {e.Message}");
+                        var createTranscriptionSBConnectionString = CreateTranscriptionEnvironmentVariables.CreateTranscriptionServiceBusConnectionString;
+                        ServiceBusUtilities.SendServiceBusMessageAsync(createTranscriptionSBConnectionString, message, log, 2).GetAwaiter().GetResult();
+                        return string.Empty;
+                    }
+
+                    var audioLengthInSeconds = 0;
+
+                    if (CreateTranscriptionEnvironmentVariables.CreateReceiptFile)
+                    {
+                        if (byteArray == null)
+                        {
+                            byteArray = await StorageUtilities.DownloadFileFromSAS(audioFileUrl).ConfigureAwait(false);
+                        }
+
+                        var audioDetails = StorageUtilities.GetAudioDetails(
+                            byteArray,
+                            audioFileName,
+                            locale,
+                            transcriptionCreationTime,
+                            modelIds.Any(),
+                            CreateTranscriptionEnvironmentVariables.AddSentimentAnalysis,
+                            CreateTranscriptionEnvironmentVariables.AddEntityRedaction,
+                            log);
+                        audioLengthInSeconds = (int)audioDetails.AudioLength.TotalSeconds;
+                        await StorageUtilities.WriteTranscriptionReceiptToStorageAsync(CreateTranscriptionEnvironmentVariables.AzureWebJobsStorage, CreateTranscriptionEnvironmentVariables.ReceiptsContainer, audioDetails, log).ConfigureAwait(false);
+                        log.LogInformation("Receipt created for flie: " + jobName);
+                    }
+
+                    var reenqueueingTimeInSeconds = MapDatasetSizeToReenqueueingTimeInSeconds(CreateTranscriptionEnvironmentVariables.AudioDatasetSize);
+
+                    // Increase delay if audio file is very short:
+                    var initialDelayInSeconds = audioLengthInSeconds <= 10 ? reenqueueingTimeInSeconds : audioLengthInSeconds;
```

Comment:
What happens if the delay is timed out?

Reply:
that's just the delay until the message get's added to the queue again. 

Anyyway, this part is removed and set to a hard value now.